### ISSUE TITLE
Loading the ".npy" files such as "cat-pikachiu.npy" or "shiba-haru-1.npy" has been failed

### ISSUE THREAD

**Issue by AHHHZ975** [facebookresearch/banmo#72]
Hi Gengshan!
Thank you for publically making available the great work you have done. 

I have been trying to reproduce the results for the task "motion retargeting" using the bash scripts you have already shared. But, I keep getting the error **"AttributeError: 'DataStore' object has no attribute 'fast_hash'"** which is coming from the function **"pickle.load()"** as can be seen in the picture below.

![image](https://github.com/facebookresearch/banmo/assets/47984138/5253bc33-478b-4526-8a8f-7c954be47859)

So, when I went through the code, I found that during the motion retargeting task, the problem happens only if the code wants to load the "**shiba-haru-1.npy**". Then, I thought maybe I was doing something wrong and therefore I tried to manually load the file using **"numpy.load()"**, and again I got the same error. So, I tried to load other pre-trained models like **"cat-coco.npy"** or **"cat-pikachiu.npy"** as well and I got the same issue for them too. 

I should also mention that I can successfully load all the **".pth"** files using "numpy.load()" but the problem happens only when trying to load the **".npy"** files.

I would appreciate it if you could help me out with this. Thank you in advance.

Kind Regards,
Amir

**Comment by AHHHZ975** [comment]
NOTE: I should also mention that I tried this using different versions of numpy, pickle, and python on both google colab and my local computer and I keep getting the same error every single time.

**Comment by gengshan-y** [comment]
Hi Amir, I just tried to load the [cat-coco.npy](https://www.dropbox.com/s/tvc9rpef9fgb7vp/cat-coco.npy
) file, and it works as expected.


Besides numpy arrays, what's inside the .npy file includes a trimesh object, and a TrackedArray. Perhaps a different version of trimesh is causing the issue?

Here's the one I was using
```
$pip list | grep trimesh
trimesh                       3.9.43
```

**Comment by AHHHZ975** [comment]
Hi Gengshan, thank you very much for the reply, and sorry that I am replying late. Yes, the problem was the versioning of the trimesh. Also, after changing the version of the trimesh, I faced similar issues regarding the libraries like Imageio and imageio-ffmpeg. Surprisingly, some attributes in some functions have been replaced by others and that is why I got several errors. But, fortunately, solved and could successfully re-implement different tasks. Thanks again for your help!

Kind Regards,
Amir

Kind Regards,
Amir
